docs/source/fetching/persisted-queries.mdx

```diff
@@ -0,0 +1,160 @@
+---
+title: Persisted Queries
+---
+
+Apollo supports two separate but related features called **persisted queries** and **automatic persisted queries** (APQs).
+With both features, clients can execute a GraphQL operation by sending an operation's ID instead of the entire string. An operation's ID is the SHA256 hash of the full operation string. Querying by ID can significantly reduce latency and bandwidth usage for very large operation strings.
+
+## Differences between persisted queries and APQ
+
+The persisted queries feature requires operations to be preregistered in a **persisted query list** (**PQL**). 
+This allows the PQL to act as a safelist of trusted operations made by your first-party apps. As such, persisted queries is a security feature as much as a performance one.
+
+With APQs, if the server can't find the operation ID the client provides, the server returns an error indicating that it needs the full operation string. If the Apollo iOS client receives this error, it automatically retries the operation with the full operation string.
+
+If you _only_ want to improve request latency and bandwidth usage, APQ addresses your use case. If you _also_ want to secure your supergraph with operation safelisting, you should preregister trusted operations in a PQL.
+
+For more details on differences between persisted queries and APQ, see the [GraphOS persisted queries documentation](/graphos/operations/persisted-queries#differences-from-automatic-persisted).
+
+### Requirements
+
+Persisted queries is currently in [preview](/resources/product-launch-stages#preview) and has the following requirements:
+- Apollo iOS (v1.3.1+)
+- [Apollo Router](/router) (v1.25.0+)
+- [GraphOS Enterprise plan](/graphos/enterprise/)
+
+You can use APQ with the following versions of Apollo iOS, Apollo Router, and Apollo Server:
+- Apollo iOS (v1.0.0+)
+- [Apollo Router](/router) (v0.1.0+)
+- [Apollo Server](/apollo-server/) (v1.0.0+)
+
+## Implementation steps
+
+Both persisted queries and APQ require you to configure code generation and how your client makes requests. The rest of this article details these configurations.
```

Comment:
Nit: only only PQs require to generate a manifest. APQs work out of the box, at least on Android, there's no codegen configuration to do at all